crates/iroha_data_model/src/transaction.rs

```diff
@@ -182,7 +183,7 @@ mod model {
 
 impl<A: Instruction> FromIterator<A> for Executable {
     fn from_iter<T: IntoIterator<Item = A>>(iter: T) -> Self {
-        Self::Instructions(iter.into_iter().map(Into::into).collect())
+        Self::Instructions(iter.into_iter().map(Into::into).collect::<Vec<_>>().into())
```

Comment:
```suggestion
        Self::Instructions(iter.into_iter().map(Into::into).collect())
```

can we implement FromIterator for ConstVec?